packages/experimental/Checkbox/src/CheckboxTokens.macos.ts

```diff
@@ -0,0 +1,31 @@
+import { Theme } from '@fluentui-react-native/framework';
+import { TokenSettings } from '@fluentui-react-native/use-styling';
+import { CheckboxTokens } from '.';
+
+export const checkboxStates: (keyof CheckboxTokens)[] = ['disabled', 'boxAtEnd', 'checked'];
+
+export const defaultCheckboxTokens: TokenSettings<CheckboxTokens, Theme> = (t: Theme) =>
+  ({
+    checkboxBackgroundColor: 'transparent',
+    textBorderColor: 'transparent',
+    checkboxBorderColor: t.colors.checkboxBorderColor,
+    checkmarkColor: t.colors.checkmarkColor,
+    checkmarkVisibility: 0,
+    variant: 'bodyStandard',
+    checkboxMarginStart: 0,
+    checkboxMarginEnd: 5,
+    color: t.colors.menuItemText,
+    disabled: {
+      checkboxBackgroundColor: t.colors.checkboxBackgroundDisabled,
+      checkboxVisibility: 0.38,
+    },
+    checked: {
+      checkboxBorderColor: t.colors.checkboxBackground,
+      checkboxBackgroundColor: t.colors.checkboxBackground,
+      checkmarkVisibility: 1,
+    },
+    boxAtEnd: {
+      checkboxMarginStart: 4,
+      checkboxMarginEnd: 0,
+    },
+  } as CheckboxTokens);
```

Comment:
Are the macOS tokens and styling actually different? Or are they inherited from the old checkbox? or did you choose them from Figma?

Reply:
I couldn't find any Figma files for checkbox on macOS. If you know where they exist, send them my way :) 

The values for the tokens on macOS are slightly different. I pulled them from both the old checkbox and this file: https://github.com/microsoft/fluentui-react-native/blob/master/packages/theming/apple-theme/src/appleTheme.macos.ts#L49 which the old checkbox seemed to be picking up the styles from but I'm not sure how to hook it up to the new framework so I copied them over directly. @Saadnajmi do you know how this file works/if I can get it to work for the new checkbox?

Reply:
https://www.figma.com/file/RKqH3UfoIZpO1ZZIlMSp0j/Fluent-vNext-macOS?node-id=1406%3A2386
I wouldn't give it too much weight though, designs' take for most macOS controls has been "use the system control".
Those values I think Tushar added, but are just the same structure as a settings file for the old framework. In effect, they override the default settings of the control. I used to put control specific overrides there because I thought all the "theme" info should be in the apple theme files instead of in a .<platform> file. It feels like we're not really doing that anymore, especially with the token pipeline.

The settings aren't being picked up from the apple theme because the names are different: "checkbox" vs "Checkbox". 
I kinda like having them not picked up though, it kinda feels like we don't want to do the thing where we put component settings in that one file, so I wouldn't change how you've done it =)

Reply:
Actually, wait, both are "Checkbox". I'm not sure why those settings aren't messing with the settings here. Happy accident?